### PR TITLE
Added engine-cleanup role

### PR DESCRIPTION
**Pull request by lukas-bednar**

### CODE REVIEW

**Comment by lukas-bednar** [comment]
@StLuke @pkubica

File: roles/ovirt-engine-cleanup/templates/answerfile_4.0.txt.j2 [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+OVESETUP_CONFIG/firewallManager=str:firewalld
+OVESETUP_CONFIG/remoteEngineHostRootPassword=none:None
+OVESETUP_CONFIG/remoteEngineHostSshPort=none:None
+OVESETUP_CONFIG/fqdn=str:{{ ovirt_engine_hostname }}
+OVESETUP_REMOVE/confirmUninstallGroups=bool:True
+OVESETUP_REMOVE/removeOptions=multi-str:ovirt-imageio-proxy
+ovirt-engine-websocket-proxy
+OVESETUP_REMOVE/enabledFileGroups=str:,core,ca_config,iso_images,ovirt_dwh_files,vmconsole_proxy_helper_pki,iso_domain,ca_pki,vmconsole_proxy_pki,exportfs,versionlock,ssl,nfs_config
+OVESETUP_REMOVE/removeAll=bool:True
+OVESETUP_REMOVE/removeChanged=none:None
```
Comment (lukas-bednar) [inline roles/ovirt-engine-cleanup/templates/answerfile_4.0.txt.j2]: True?

Comment (StLuke) [inline roles/ovirt-engine-cleanup/templates/answerfile_4.0.txt.j2]: True,

We should also consider cleaning up packages, /etc, /var/lib, /usr/share, I would consider these to be disabled by default and clear them only if some variable is set

Comment (pkubica) [inline roles/ovirt-engine-cleanup/templates/answerfile_4.0.txt.j2]: +1

**Comment by pkubica** [comment]
I think we can merge it soon.

File: roles/ovirt-engine-cleanup/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ovirt_engine_dwh: True
+ovirt_engine_type: 'ovirt-engine'
+ovirt_engine_version: '4.0'
+ovirt_engine_admin_password: '123456'
+
+ovirt_engine_db_host: 'localhost'
+ovirt_engine_db_port: 5432
+ovirt_engine_db_name: 'engine'
+ovirt_engine_db_user: 'engine'
+ovirt_engine_db_password: 'AqbXg4dpkbcVRZwPbY8WOR'
```
Comment (StLuke) [inline roles/ovirt-engine-cleanup/defaults/main.yml]: dont put generated passwords here

File: roles/ovirt-engine-cleanup/defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+ovirt_engine_dwh_db_host: 'localhost'
+ovirt_engine_dwh_db_port: 5432
+ovirt_engine_dwh_db_name: 'ovirt_engine_history'
+ovirt_engine_dwh_db_user: 'ovirt_engine_history'
+ovirt_engine_dwh_db_password: '37xmBKECANQGm0z3SfylMp'
```
Comment (StLuke) [inline roles/ovirt-engine-cleanup/defaults/main.yml]: dont put generated passwords here

File: roles/ovirt-engine-cleanup/templates/answerfile_4.0.txt.j2 [hunk 4]
```diff
@@ -0,0 +1,49 @@
+[environment:default]
+OVESETUP_CORE/remove=bool:True
+OVESETUP_CONFIG/remoteEngineSetupStyle=none:None
+OVESETUP_CONFIG/sanWipeAfterDelete=bool:False
+OVESETUP_CONFIG/firewallManager=str:firewalld
+OVESETUP_CONFIG/remoteEngineHostRootPassword=none:None
+OVESETUP_CONFIG/remoteEngineHostSshPort=none:None
+OVESETUP_CONFIG/fqdn=str:{{ ovirt_engine_hostname }}
```
Comment (StLuke) [inline roles/ovirt-engine-cleanup/templates/answerfile_4.0.txt.j2]: {{ ansible_fqdn }}

File: roles/ovirt-engine-cleanup/meta/main.yml [hunk 5]
```diff
@@ -0,0 +1,47 @@
+galaxy_info:
+  author: Lukas Bednar
+  description: generates answer file for engine-cleanup and execute it.
+  company: Red Hat
+
+  # If the issue tracker for your role is not on github, uncomment the
+  # next line and provide a value
+  issue_tracker_url: https://github.com/StLuke/ovirt-ansible/issues
```
Comment (StLuke) [inline roles/ovirt-engine-cleanup/meta/main.yml]: not in my repositories anymore (ownership was moved from StLuke)

File: roles/ovirt-engine-cleanup/README.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+Dependencies
+------------
+
+None
+
+Example Playbook
+----------------
+
+TODO
```
Comment (StLuke) [inline roles/ovirt-engine-cleanup/README.md]: TODO? This needs to be fixed before the merge

Comment (pkubica) [inline roles/ovirt-engine-cleanup/README.md]: Yes, you are right. I miss that.

**Comment by pkubica** [comment]
done via #96

**Pull request closed without merging**